compose.yaml

```diff
@@ -64,17 +61,23 @@ services:
       DATABASE_HOST: "${COMPOSE_PROJECT_NAME}-db"
       DATABASE_PASSWORD: ${DATABASE_PASSWORD:-secret}
       DATABASE_PORT: "5432"
+      DJANGO_SUPERUSER_EMAIL: ${DJANGO_SUPERUSER_EMAIL:-dev@test.com}
+      DJANGO_SUPERUSER_PASSWORD: ${DJANGO_SUPERUSER_PASSWORD:-secret}
     env_file: .env
     networks:
       - ${COMPOSE_PROJECT_NAME}
     ports:
-      - "8000:8000"
+      - "127.0.0.1:8000:8000"
     volumes:
       - "./:/${COMPOSE_PROJECT_NAME}"
     working_dir: "/${COMPOSE_PROJECT_NAME}"
 
 networks:
   wevote:
+    ipam:
+      config:
+        - subnet: "172.27.0.0/16"
```

Comment:
Why not let docker figure out the subnet to use? If this particular subnet is already defined for another container, it will fail to work.